engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -21,41 +21,43 @@
 import org.terasology.config.RenderingConfig;
 import org.terasology.context.Context;
 import org.terasology.rendering.opengl.AbstractFBOsManager;
-import org.terasology.rendering.opengl.DefaultDynamicFBOs;
-import static org.terasology.rendering.opengl.DefaultDynamicFBOs.FINAL;
-import static org.terasology.rendering.opengl.DefaultDynamicFBOs.READ_ONLY_GBUFFER;
-import static org.terasology.rendering.opengl.DefaultDynamicFBOs.WRITE_ONLY_GBUFFER;
+import static org.terasology.rendering.opengl.ScalingFactors.FULL_SCALE;
+
 import org.terasology.rendering.opengl.FBO;
 import org.terasology.rendering.opengl.FBOConfig;
 import org.terasology.rendering.opengl.ScreenGrabber;
+import org.terasology.rendering.world.WorldRenderer;
+import org.terasology.rendering.world.WorldRendererImpl;
 
 /**
  * TODO: Add javadocs
  * TODO: Better naming
  */
 public class DisplayResolutionDependentFBOs extends AbstractFBOsManager {
+    public static final ResourceUrn SCENE_OPAQUE = new ResourceUrn("engine:sceneOpaque");
+    public static final ResourceUrn SCENE_OPAQUE_PING_PONG = new ResourceUrn("engine:sceneOpaquePingPong");
+    public static final ResourceUrn SCENE_FINAL = new ResourceUrn("engine:sceneFinal");
+
     private FBO.Dimensions fullScale;
     private RenderingConfig renderingConfig;
     private ScreenGrabber screenGrabber;
+    Context context;
 
     public DisplayResolutionDependentFBOs(Context context) {
+        this.context = context;
```

Comment:
Can you please check if we can pass the renderingConfig and the screenGrabber directly to the constructor? We might not need to pass the context anymore.